app/views/posts/show.html.erb

```diff
@@ -1,19 +1,18 @@
 <div class="post-container">
-   <div class="post-header">
-       <h2><%= @post.title%> By <%= @post.author.name%></h2>
-         <ul>
-             <li>Comments:<%= @post.comments_counter%></li>
-             <li>Likes:<%=@post.likes_counter%></li>
-         </ul>
-   </div>
-    <h4 class="comment-title"> Comments</h4>
-    <div class="post-body">
-       <%=@post.text%>
-    </div>
-     <div class="comment-container">
-         <% @post.comments.each do |comment| %>
-     <span><%= comment.author.name if comment.author %>: <%= comment.text %></span>
-   <% end %>
-
-     </div>
-</div>
\ No newline at end of file
+  <div class="post-header">
+    <h2><%= @post.title %> By <%= @post.author.name %></h2>
+    <ul>
+      <li>Comments: <%= @post.comments.count %></li>
+      <li>Likes: <%= @post.likes.count %></li>
+    </ul>
+  </div>
+  <h4 class="comment-title">Comments</h4>
+  <div class="post-body">
+  <%= @post.text %>
+</div>
+  <div class="comment-container">
+    <% @post.comments.each do |comment| %>
+      <span><%= comment.author.name if comment.author %>: <%= comment.text %></span>
+    <% end %>
+  </div>
```

Comment:
- [ ] Kindly add some kind of button that allows the user to like a post, as there is currently no way for a user to add a like on your app.

  ![image](https://github.com/AlyAtalla/blog-app/assets/36542986/bf91901c-545d-4955-a79d-3bc1d08de4cb)